dc_logging_aws/constructs/tasks/postcode_searches_query.py

```diff
@@ -69,17 +73,11 @@ def __init__(
                 }
             ),
             query_language=sfn.QueryLanguage.JSONATA,
-        )
-
-        result_task = tasks.AthenaGetQueryResults(
-            self,
-            f"Get {task_name} Result",
-            query_execution_id="{% $states.input.Payload.queryExecutionId %}",
-            query_language=sfn.QueryLanguage.JSONATA,
             assign={
-                result_variable_name: "{% $states.result.ResultSet.Rows[1].Data[0].VarCharValue %}"
+                result_variable_name: "{% $states.result.Payload.queryExecutionId %}"
```

Comment:
The reason for this change is that there's a limit to how many bytes the `AthenaGetQueryResults` can handle, and we were going over this. However the idea that we can do something with the results, as part of this construct, seems good to me. At the moment I'm just assigning the `queryExecutionId` to the query name so that they're easy to gather up and go find after running the state machine.